modules/python3qt/include/modules/python3qt/pythoneditorwidget.h

```diff
@@ -57,17 +57,17 @@ class SyntaxHighligther;
 class CodeEdit;
 
 class IVW_MODULE_PYTHON3QT_API PythonEditorWidget : public InviwoDockWidget,
-                                                    public PythonExecutionOutputObeserver {
-
+                                                    public PythonExecutionOutputObserver {
 public:
-    PythonEditorWidget(QWidget* parent, InviwoApplication* app);
+    PythonEditorWidget(QWidget* parent, InviwoApplication* app,
+                       std::function<void(const std::string&)> callback = nullptr);
```

Comment:
better name for `callback`? Or add documentation that this refers to the text changed callback.

Reply:
Especially since setting this callback changes the behavior of the dialog.